backend/src/main/java/fk/prof/backend/aggregator/ProfileWorkInfo.java

```diff
@@ -98,7 +98,7 @@ private int getSampleCount(Recorder.Wse wse) {
   protected FinalizedProfileWorkInfo buildFinalizedEntity() {
     Map<AggregatedProfileModel.WorkType, Integer> mappedWorkTypeSamples = new HashMap<>();
     for(Map.Entry<Recorder.WorkType, Integer> entry: workTypeSamples.entrySet()) {
-      AggregatedProfileModel.WorkType mappedWorkType = ProtoUtil.mapWorkType(entry.getKey());
+      AggregatedProfileModel.WorkType mappedWorkType = ProtoUtil.mapRecorderToAggregatorWorkType(entry.getKey());
       if(mappedWorkType == null) {
```

Comment:
I wanted a more descriptive exception to be thrown rather than a NullPointerException. Presently, the exception will indicate which work type was failed to match, looking at logs makes the mistake clearer to a dev